### PR TITLE
Refactor(Order, Finance)

### PR DESCRIPTION
**Pull request by meeklife**

### CODE REVIEW

File: apps/finance/views.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from rest_framework import mixins
+from rest_framework.viewsets import GenericViewSet
+
+from .models import Transaction
+from .serializers import TransactionSerializer
+
+
+class Transactions(mixins.ListModelMixin, mixins.RetrieveModelMixin, GenericViewSet):
```
Comment (kryzbone) [inline apps/finance/views.py:8]: Also only admins should have access to all transaction.. other users should only see their own transactions

File: apps/orders/views.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+class Orders(
+    mixins.UpdateModelMixin,
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+    GenericViewSet,
+):
+    queryset = Order.objects.all()
+    serializer_class = OrderSerializer
+
```
Comment (kryzbone) [inline apps/orders/views.py:16]: Same thing here... Only admins should see all orders other users should only see their orders and only their orders

File: apps/orders/models.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+from apps.inventory.models import Inventory
+from apps.users import models as usermodels
+
+
+class Order(base_models.BaseModel):
+    class Order_Status(models.TextChoices):
+        DRAFT = (
+            "D",
+            _("Draft"),
+        )
```
Comment (kryzbone) [inline apps/orders/models.py:15]: This is just a suggestion... why is this not on one line like this `DRAFT = "D", _("Draft")`

Comment (meeklife) [reply apps/orders/models.py:15]: The isort cmd is refactoring it when I save it

Comment (kryzbone) [reply apps/orders/models.py:15]: remove the outer brackets.. that should fix that

**Review by kryzbone — APPROVED** [review]

**Pull request merged**